relay-event-normalization/src/normalize/mod.rs

```diff
@@ -349,7 +351,7 @@ fn remove_invalid_measurements(
         }
 
         // Check if this is a builtin measurement:
-        for builtin_measurement in &measurements_config.builtin_measurements {
+        for builtin_measurement in &builtin_measurement_keys {
```

Comment:
changed to a vec because this loop is called on each iteration, so the iterator would be exhausted after the first one. is there a better way to do it ? @iker-barriocanal

Reply:
I've been thinking about it and can't find any better solutions, unfortunately. Let's see if other reviewers have better suggestions.

Reply:
One way you could solve this is by passing the view type in here. You've already created `DynamicMeasurementsConfig` and expose iterators. So you can pass this in and iterate them explicitly here.

Reply:
Instead of collecting to a vec, just call the iterator every time here.